core/src/main/java/com/netflix/iceberg/util/BinPacking.java

```diff
@@ -23,6 +23,7 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Lists;
+import java.util.Comparator;
```

Comment:
Do we still need this import?

Reply:
Apologies - I was incorporating @rdsr's feedback and pushed incorrectly. Currently, I've only pushed changes after rebasing with master.